source/elements/dpcpp/source/index.rst

```diff
@@ -49,151 +48,99 @@ some additional extensions being required, and some DPC++ extensions no longer
-   `Group algorithms`_          DPC++ extension           Required              Required              Not required [#tmp]_  NA [#na]_
-   `Group mask`_                DPC++ extension           Not required [#tmp]_  Not required [#tmp]_  Not required [#tmp]_  NA [#na]_
-   `Parallel for shortcuts`_    DPC++ extension           Required              Required              Required              NA [#na]_
-   `Pinned memory property`_    DPC++ extension           Required              Required              Required              NA [#na]_
-   `Reductions`_                DPC++ extension           Required [#redc]_     Required [#redc]_     Not required [#tmp]_  NA [#na]_
+   `Extended atomics`_          DPC++ extension           Required              Required              Required              NA [#na]_
+   `Filter selector`_           DPC++ extension           Required              Required              Required              NA [#na]_
+   `FPGA LSU controls`_         DPC++ extension           Not required          Not required          Required              NA [#na]_
+   `FPGA memory channel`_       DPC++ extension           Not required          Not required          Required              NA [#na]_
+   `FPGA register`_             DPC++ extension           Not required          Not required          Required              NA [#na]_
+   `FPGA selector`_             DPC++ extension           Required              Required              Required              NA [#na]_
+   `GPU device info`_           DPC++ extension           Required              Required              Required              NA [#na]_
+   `Level zero backend`_        DPC++ extension           Not required          Not required          Not required          NA [#na]_
+   `Local memory allocations`_  DPC++ extension           Required              Required              Required              NA [#na]_
+   `Pinned memory property`_             DPC++ extension           Required              Required              Required              NA [#na]_
+   `Platform context`_          DPC++ extension           Required              Required              Required              NA [#na]_
    `Restrict all arguments`_    DPC++ extension           Required              Required              Required              NA [#na]_
-   `Static local mem query`_    DPC++ extension           Not required [#tmp]_  Not required [#tmp]_  Not required [#tmp]_  NA [#na]_
-   `Sub-groups`_                DPC++ extension           Required              Required              Not required          `sub_group <https://github.com/intel/llvm/tree/sycl/sycl/test/sub_group>`__
-   `Sub-group algorithms`_      DPC++ extension           Required [#sga]_      Required [#sga]_      Not required          `sub_group <https://github.com/intel/llvm/tree/sycl/sycl/test/sub_group>`__
+   `Sub-group mask`_            DPC++ extension           Required              Required              Not required          NA [#na]_
```

Comment:
Is this correct that sub-group mask is not required for FPGA?  Should this be "Likely to be required in the future"?

Reply:
I changed to required, since I'd expect it to work for the degenerate SG size of 1 which FPGA runs with